src/main/java/com/google/sps/servlets/Caption.java

```diff
@@ -29,7 +29,7 @@ public String getCaptionFromId(String videoId) {
         continue;
       }
     }
-    return null;
+    return "";
```

Comment:
Off topic: Does captionService.fetchStream(...) throw an error if the video id is not valid? I would find it out if this method return an empty string for an invalid video because it lets the caller think that its a valid video without captions.

Reply:
fetchStream throws a FileNotFound error if the video Id is invalid or the caption does not exist. So there isn't a way to differentiate that from this perspective.

Reply:
^^ building off this, CommentService throws an IllegalArgumentException if the video can't be found and when SentimentServlet catches that it'll send an error to the client and stop the servlet from running